wic/src/components/ResoruceMap.js

```diff
@@ -0,0 +1,51 @@
+import React, {PropTypes, Component} from 'react';
+import ReactDOM from 'react-dom';
+
+let map;
+
+export default class ResoruceMap extends Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    const geocode_options = {
+      address: this.props.zipcode || '94010'
+    }
+    new window.google.maps.Geocoder().geocode(geocode_options, (results, status) => {
+      if (status == 'OK') {
```

Comment:
Also, I'm a big of returning from errors immediately instead of nesting logic:
https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style#Return_from_errors_immediately

Reply:
eating the errors for now, this will keep the map from appearing.  maybe in the next iteration we can come-up with a more elegant way to bubble the errors up.